model/task_lifecycle.go

```diff
@@ -732,12 +732,15 @@ func updateDisplayTask(t *task.Task) error {
 }
 
 type RestartTaskOptions struct {
-	DryRun     bool      `bson:"dry_run" json:"dry_run" yaml:"dry_run"`
-	OnlyRed    bool      `bson:"only_red" json:"only_red" yaml:"only_red"`
-	OnlyPurple bool      `bson:"only_purple" json:"only_purple" yaml:"only_purple"`
-	StartTime  time.Time `bson:"start_time" json:"start_time" yaml:"start_time"`
-	EndTime    time.Time `bson:"end_time" json:"end_time" yaml:"end_time"`
-	User       string    `bson:"user" json:"user" yaml:"user"`
+	DryRun    bool      `bson:"dry_run" json:"dry_run"`
+	StartTime time.Time `bson:"start_time" json:"start_time"`
+	EndTime   time.Time `bson:"end_time" json:"end_time"`
+	User      string    `bson:"user" json:"user"`
+
+	// note that the bson tags are not quite accurate, but are kept around for backwards compatibility
```

Comment:
The json tags don't match what you're using in the UI.
Also I don't feel like this comment is necessary, but I don't really care if you leave it in or not.

Reply:
They don't, but they aren't used in the UI